packages/core/src/GondelDomUtils.ts

```diff
@@ -10,6 +15,24 @@ function isElement(domNode: ArrayLikeHtmlElement): domNode is Element {
   return (domNode as Element).nodeType !== undefined;
 }
 
+/**
+ * Inspired by the RXJS anchor approach by using symbols (if supported) or strings
+ * for internal fixtures.
+ *
+ * @param {string=g} namespace
+ * @param {string?} addition
+ * @see https://github.com/ReactiveX/rxjs/blob/master/src/internal/symbol/rxSubscriber.ts
+ */
+export function getGondelAttribute(namespace: string = "g", addition?: string): string {
+  const id = `__gondel_${addition ? addition + "_" : ""}${namespace}__`;
+
+  if (Symbol && typeof Symbol.for === "function") {
```

Comment:
Can we remove the Symbol here?
It doesn't prevent access and will reduce development experience.